arch/cpu/nrf/Makefile.nrf

```diff
@@ -73,9 +75,67 @@ NRFX_INC_PATHS += templates
 NRFX_INC_PATHS += drivers/src
 NRFX_INC_PATHS += drivers/include
 NRFX_INC_PATHS += soc
+NRFX_INC_PATHS += hal
 
 EXTERNALDIRS += $(addprefix $(NRFX_ROOT)/, $(NRFX_INC_PATHS))
 
+## Tinyusb
+ifdef TINYUSB
+  CFLAGS += -DCFG_TUSB_MCU=OPT_MCU_NRF5X
+  #nrf specific
+  TINYUSB_C_SRCS += $(TINYUSB_ROOT)/src/portable/nordic/nrf5x/dcd_nrf5x.c
+
+  TINYUSB_INC_PATHS += src/portable/nordic/nrf5x/
+
+  #source common
+  TINYUSB_C_SRCS += $(TINYUSB_ROOT)/src/tusb.c
+  TINYUSB_C_SRCS += $(TINYUSB_ROOT)/src/common/tusb_fifo.c
+  TINYUSB_C_SRCS += $(TINYUSB_ROOT)/src/device/usbd.c
+  TINYUSB_C_SRCS += $(TINYUSB_ROOT)/src/device/usbd_control.c
+  TINYUSB_C_SRCS += $(TINYUSB_ROOT)/src/class/audio/audio_device.c
```

Comment:
Is there a reason for building unused classes?

Reply:
I think this was just leftover. I'll remove all classes except cdc which is the only one being used.